src/BloomExe/Edit/EditingView.cs

```diff
@@ -1394,7 +1394,14 @@ public static string GetInstructionsForUnlockingBook()
 		public int Zoom
 		{
 			// Whatever the user may have saved (e.g., from earlier use of ctrl-wheel), we'll make this an expected multiple-of-10 percent.
-			get { return (int) Math.Round(float.Parse(Settings.Default.PageZoom ?? "1.0") * 10) * 10; }
+			get
```

Comment:
This one loses the tolerance of a messed up settings value.

How about instead doing something like

    if(float.TryParse(...))  return (int) Math....
    else return 100;